src/main/java/ai/rev/languageid/models/LanguageIdJobOptions.java

```diff
@@ -0,0 +1,196 @@
+        this.notificationConfig = new CustomerUrlData(callbackUrl, authHeaders);
+    }
+
+    /**
+     * Optional property to specify the callback url that Rev AI will POST to when job processing is complete
+     *
+     * @param callbackUrl The url to POST to when job processing is complete.
+     */
+    public void setNotificationConfig(String callbackUrl) {
+        setNotificationConfig(callbackUrl, null);
+    }
+
+    /**
+     * Returns the metadata.
+     *
+     * @return A String that contains the metadata.
+     */
+    public String getMetadata() {
+        return metadata;
+    }
+
+    /**
+     * Optional metadata that is provided during job submission limited to 512 characters.
+     *
+     * @param metadata A String to set as the metadata.
+     */
+    public void setMetadata(String metadata) {
+        this.metadata = metadata;
+    }
+
+    /**
+     * Returns the value of deleteAfterSeconds.
+     *
+     * @return The deleteAfterSeconds value.
+     */
+    public Integer getDeleteAfterSeconds() {
+        return deleteAfterSeconds;
+    }
+
+    /**
+     * Specifies the number of seconds to be waited until the job is auto-deleted after its
+     * completion.
+     *
+     * @param deleteAfterSeconds The number of seconds after job completion when job is auto-deleted.
+     */
+    public void setDeleteAfterSeconds(Integer deleteAfterSeconds) {
+        this.deleteAfterSeconds = deleteAfterSeconds;
+    }
+
+    /**
+     * Returns the media url.
+     *
+     * @return The media url.
+     * @deprecated Set sourceConfig and use getSourceConfig instead
+     */
+    @Deprecated
+    public String getMediaUrl() {
+        return mediaUrl;
+    }
+
+    /**
+     * Specifies the url where the media can be downloaded.
+     *
+     * @param mediaUrl The direct download url to the file.
+     * @deprecated Use setSourceConfig instead
+     */
+    @Deprecated
+    public void setMediaUrl(String mediaUrl) {
+        this.mediaUrl = mediaUrl;
+    }
+
+    /**
+     * Returns the source config object.
+     *
+     * @return the source config.
+     */
+    public CustomerUrlData getSourceConfig() {
+        return this.sourceConfig;
+    }
+
+    /**
+     * Specifies the url and any optional auth headers to access the source media download url.
+     *
+     * @param sourceMediaUrl The direct download url to the file.
+     * @param sourceAuth     The auth headers to the source media download url.
+     */
+
+    public void setSourceConfig(String sourceMediaUrl, Map<String, String> sourceAuth) {
+        this.sourceConfig = new CustomerUrlData(sourceMediaUrl, sourceAuth);
+    }
+
+    /**
+     * Specifies the source media download url.
+     *
+     * @param sourceMediaUrl The direct download url to the file.
+     */
+
+    public void setSourceConfig(String sourceMediaUrl) {
+        this.sourceConfig = new CustomerUrlData(sourceMediaUrl, null);
```

Comment:
```suggestion
        setSourceConfig(sourceMediaUrl, null);
```
Technically this should be done this way so theres a single source of truth method.